### DISCUSSION TITLE
add the "install the extension" section to the React guide

### DISCUSSION THREAD

**Discussion by yanggggjie** [crxjs/chrome-extension-tools#709] [Q&A]
Hi, jack👋
I'm a complete newbie and I'm a bit lost following the React documentation.
In the "opening the extension" section, I couldn't find the extension because I didn't realize I needed to follow the "install the extension" section first. 
Would you be willing to add the "install the extension" section to the React guide as well? Thank you so much.

![image](https://user-images.githubusercontent.com/60206592/236670973-2931dd15-3b4b-4a07-8f7a-dfdeb2fba8ca.png)

**Comment by michal-mazurkiewicz** [comment]
I believe it would be also nice to give some more info about building production version as I have currently huge problem with that.

**Reply by Toumash** [reply]
What was the problem?

**Comment by Toumash** [comment]
I've added a PR adding the missing section https://github.com/crxjs/chrome-extension-tools/pull/771
